### ISSUE TITLE
Power prediction

### ISSUE THREAD

**Issue by srennug** [openclimatefix/uk-wind-modelling-basic#3]
## Detailed Description
The code attached predicts the power generated at the wind farm, using the trueU V wind speed at the location.

## Context
I'm having trouble rescaling the power values but thought I would post this while I'm fixing it. 

This is the closest I've got my predicted values and it came from using a conv layer instead of an lstm. 

The data I've pulled so far is 11 months at 6H intervals. The Elexon database has no problem pulling data at shorter periods but the wind speeds give errors when setting lower periods. Extending the training data for longer than a year should help with accuracy. 

## Possible Implementation
I have a model that predicts the U V wind speeds which once I've improved could be used as data for the power predictions. 

[windspeeds_6H2021.03 copy.txt](https://github.com/openclimatefix/uk-wind-modelling-basic/files/12861872/windspeeds_6H2021.03.copy.txt)
[Wind_gen copy.txt](https://github.com/openclimatefix/uk-wind-modelling-basic/files/12861874/Wind_gen.copy.txt)
[power_predict copy.txt](https://github.com/openclimatefix/uk-wind-modelling-basic/files/12861964/power_predict.copy.txt)

**Comment by peterdudfield** [comment]
Thanks @srennug for doing this - looks great. Ill have a proper look at this soon, and we can chat next steps

**Comment by srennug** [comment]
[power_predict copy 2.txt](https://github.com/openclimatefix/uk-wind-modelling-basic/files/12870014/power_predict.copy.2.txt)
this one print the graph.

**Comment by peterdudfield** [comment]
Can you drop the plot in here?

**Comment by srennug** [comment]
![Graph_V1](https://github.com/openclimatefix/uk-wind-modelling-basic/assets/95190945/7ce998a9-2f1f-4eb9-929c-360b0ae19ade)

**Comment by srennug** [comment]
![Graph_V2](https://github.com/openclimatefix/uk-wind-modelling-basic/assets/95190945/e8222e83-3eac-42de-98dc-46f31dd1f9be)

**Comment by peterdudfield** [comment]
> ![Graph_V2](https://user-images.githubusercontent.com/95190945/274379165-e8222e83-3eac-42de-98dc-46f31dd1f9be.png)

Amazing, well done. What kinda of MAE do you get?

**Comment by srennug** [comment]
Root Mean Squared Error (RMSE): 0.19993964752094398
Mean Absolute Error (MAE): 0.15029498619541223

**Comment by peterdudfield** [comment]
Nice one thanks for this. Do you have this in MW's too? 

Do you want to jump on a call to discuss these results and next steps?

**Comment by srennug** [comment]
Yeah that' be great, I haven't been able to rescale my values yet so don't have MW.

**Comment by srennug** [comment]
![graph unscaled](https://github.com/openclimatefix/uk-wind-modelling-basic/assets/95190945/9c0313e5-7cd8-4cbf-87f9-df584076c017)
Root Mean Squared Error (RMSE): 41.86889937595835
Mean Absolute Error (MAE): 30.657921006194258

This is when I don't rescale the inputs.

**Comment by srennug** [comment]
Let me know what time suits you best to discuss the next steps. I finish uni at 4 tomorrow so anytime after then works for me.
